`CircularProgress` prop deprecation (#1706)
iTwin/stratakit: .changeset/yummy-regions-pay.md

---
"@stratakit/mui": minor
---

> `CircularProgress` props `disableShrink`, `enableTrackSlot`, and `thickness` have been deprecated.

Reviewer: ```suggestion
Deprecated `disableShrink`, `enableTrackSlot` and `thickness` props of `CircularProgress`.
```
